### PR TITLE
CODE RUB: Code Cleanup ReactionsApiTests.cs #334

### PR DESCRIPTION
**Pull request by imjane5**
Simplify namespace reference: >> Added the respective library 

Explicit left-hand side: The datatype has been specified 

this qualifier: qualifier added

Implicit left-hand side: "var" datatype has been used

Remove extra line: extra line removed.

### CODE REVIEW

File: G2H.Api.Web.Tests.Acceptance/Apis/Reactions/ReactionsApiTests.cs [hunk 1]
```diff
@@ -26,16 +27,16 @@ public partial class ReactionsApiTests
         public ReactionsApiTests(ApiBroker apiBroker) =>
             this.apiBroker = apiBroker;
 
-        private async ValueTask<List<Models.Reactions.Reaction>> CreateReactionsAsync()
+        private async ValueTask<List<Reaction>> CreateReactionsAsync()
         {
-            var userId = Guid.NewGuid();
+            Guid userId = Guid.NewGuid();
             List<Web.Models.Reactions.Reaction> reactions = GetStorageReactions(userId);
```
Comment (ShriHumrudha) [inline G2H.Api.Web.Tests.Acceptance/Apis/Reactions/ReactionsApiTests.cs:33]: You can simplify this namespace as well.

File: G2H.Api.Web.Tests.Acceptance/Apis/Reactions/ReactionsApiTests.cs [hunk 2]
```diff
@@ -44,13 +45,13 @@ private async ValueTask<List<Models.Reactions.Reaction>> CreateReactionsAsync()
 
         private static List<Web.Models.Reactions.Reaction> GetStorageReactions(Guid userId)
         {
-            List<Web.Models.Reactions.Reaction> reactions =
+           var reactions =
                 new List<Web.Models.Reactions.Reaction>();
```
Comment (ShriHumrudha) [inline G2H.Api.Web.Tests.Acceptance/Apis/Reactions/ReactionsApiTests.cs:49]: You can move this line to above.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

**Comment by imjane5** [comment]
Closes #334

File: G2H.Api.Web.Tests.Acceptance/Apis/Reactions/ReactionsApiTests.cs [hunk 3]
```diff
@@ -26,16 +27,16 @@ public partial class ReactionsApiTests
         public ReactionsApiTests(ApiBroker apiBroker) =>
             this.apiBroker = apiBroker;
 
-        private async ValueTask<List<Models.Reactions.Reaction>> CreateReactionsAsync()
+        private async ValueTask<List<Reaction>> CreateReactionsAsync()
         {
-            var userId = Guid.NewGuid();
-            List<Web.Models.Reactions.Reaction> reactions = GetStorageReactions(userId);
+            Guid userId = Guid.NewGuid();
+            var reactions = GetStorageReactions(userId);
```
Comment (ShriHumrudha) [inline G2H.Api.Web.Tests.Acceptance/Apis/Reactions/ReactionsApiTests.cs:33]: This has to be explicit

File: G2H.Api.Web.Tests.Acceptance/Apis/Reactions/ReactionsApiTests.cs [hunk 4]
```diff
@@ -44,13 +45,12 @@ private async ValueTask<List<Models.Reactions.Reaction>> CreateReactionsAsync()
 
         private static List<Web.Models.Reactions.Reaction> GetStorageReactions(Guid userId)
```
Comment (ShriHumrudha) [inline G2H.Api.Web.Tests.Acceptance/Apis/Reactions/ReactionsApiTests.cs:46]: Fis this namespace as well. Verify any other occurrences too.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

**Review by ShriHumrudha — APPROVED** [review]

**Comment by cjdutoit** [comment]
Thanks @imjane5

**Review by cjdutoit — APPROVED** [review]

**Pull request merged**